docs/decisions/0007-lti-1p3-decoupling.rst

```diff
@@ -0,0 +1,115 @@
+0007 Decouple LTI 1.3 Launch from XBlock and edX Platform
```

Comment:
Does this decision really apply to only LTI 1.3? It seems like we should deciding we want these features for any launch, it's just the 1.3 implementation that is most impacted by this because of the extra steps involved.

Reply:
boot camp LTI tools turn out to be currently LTI 1.1 and probably need to be decoupled from the blocks

Reply:
best news I've heard all day 😂  (that these tools aren't 1.3/advantage).

You can kinda do a 'decoupled' 1.1 launch already but IMO we should commit to a similar interface for both even if 1.1 is simple enough to hack on from the outside.

Reply:
Just FYI - We were told yesterday that Bootcamps will be updating their integrations to LTI 1.3 before the beginning of the calendar year 2023.

Reply:
I think if it's just another couple of tickets to do 1.1 as well that would insulate us from the 1.3 transition slipping and be useful for the community

Reply:
That sounds good. I'll update the ADR to mention that we'll modify the LTI 1.1 embed function to get configuration data in the same way.